README.md

```diff
@@ -192,6 +192,22 @@ Command Options
 | `--solution=$solution`            | Set Visual Studio Solution version (win)
 
 
+Configuration
+--------
+
+#### `node-gyp` responds to environment variables:
+Variables take the form `npm_config_OPTION_NAME` for any of the options
+listed above.<br>
```

Comment:
Why the `<br>`? If you want a line break just add two trailing spaces (or leave a blank line between for a proper paragraph break). In any case I think the next line can be part of this one.